### DISCUSSION TITLE
TIL: iPad, code-server, and Kubernetes

### DISCUSSION THREAD

**Discussion by bpmct** [coder/code-server#1990] [General]
Not my article, but I love this in-depth tutorial for deploying code-server on K8s: https://brownfield.dev/post/2020-07-11-ipad-code-server-workflow/

**Comment by mterhar** [comment]
Thanks. 

I'm happy to revisit or expand.  It seems to have a pretty continuous stream of folks coming to it from google and other blogs.

**Reply by bpmct** [reply]
That would be awesome! Let us know if there is any way we can help. It looks like the copy/paste iPad fix (#1015) will be rolled out soon!
